### PR TITLE
Add footer with frontend version

### LINKED ISSUE #153: Add Version Footer to Frontend
Display Edgehog frontend version.

### PR DESCRIPTION
**Pull request by Pavinati**
Closes #153

### CODE REVIEW

File: frontend/src/components/Footer.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+footer img {
```
Comment (szakhlypa) [inline frontend/src/components/Footer.scss:1]: Let's add header with `Copyright` and `SPDX-License-Identifier` for REUSE

File: frontend/src/App.tsx [hunk 2]
```diff
@@ -35,6 +36,8 @@ import HardwareTypes from "pages/HardwareTypes";
 import Login from "pages/Login";
 import Logout from "pages/Logout";
 
+import { name, version, homepage, repository, bugs } from "../package.json";
```
Comment (szakhlypa) [inline frontend/src/App.tsx:39]: `name` is unnecessary here

File: frontend/src/components/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+const Footer = ({
+  appName,
+  appVersion,
+  homepageUrl,
+  repoUrl,
+  issueTrackerUrl,
+}: FooterProps) => {
+  return (
+    <footer className="pt-2 pb-1 border-top">
```
Comment (davidebriani) [inline frontend/src/components/Footer.tsx:43]: Why distinguish the padding `pt-2 pb-1`? I think a simpler `ph-2` would also look better.

**Review by szakhlypa — CHANGES_REQUESTED** [review]

File: frontend/src/App.tsx (hunk 2 above)
Comment (Pavinati) [reply frontend/src/App.tsx]: i was pushing the fixes at the same time as i got a notification about this comment :smile:

**Review by szakhlypa — APPROVED** [review]

File: frontend/src/components/Footer.tsx (hunk 3 above)
Comment (Pavinati) [reply frontend/src/components/Footer.tsx:43]: it was `py-2` before, but i didn't like it much when i added the top border.
It doesn't make much of a difference though

Comment (davidebriani) [reply frontend/src/components/Footer.tsx:43]: Right, I meant `py-2`. Ok, let's roll like this then

**Review by davidebriani — APPROVED** [review]

**Review by bettio — APPROVED** [review]

**Pull request merged**